Dockerfile

```diff
@@ -89,8 +89,8 @@ ARG HLS_VERSION=1.7.0.0
 RUN \
   set -o errexit -o xtrace; \
   if test -n "$HLS_VERSION"; then \
-  ghcup install hls "$HLS_VERSION" --set; \
-  ghcup gc --hls-no-ghc; \
+  ghcup compile hls --cabal-update --ghc "$GHC_VERSION" --git-ref "$HLS_VERSION"; \
+  rm --recursive /cabal-store/* ~/.cabal/logs ~/.cabal/packages; \
```

Comment:
Did you try running `haskell-language-server-wrapper` from the container this Dockerfile builds?

I recall removing `~/.cabal.packages` causing errors along the lines of `cannot find libHSbase ...`.

Reply:
I did not try that out. And this build failed in CI anyway, so I'm going to close this PR.

Reply:
I don't see any reasons for failure on your or my CI build failure output in #7. 

I was able to build it locally, so I'm guessing it's running out of memory or something.

Am I missing something or is it the same for you?

Reply:
I don't see any reason for the failure either. My guess, like yours, is out of memory. Probably `ExitFailure 9`. 

For what it's worth, it did build for me locally.